> ClickHouse special build check (actions) — 6/7 artifact groups are OK

was broken in master (fixed in https://github.com/ClickHouse/ClickHouse/pull/39288)

```
Jul 16 12:21:10 /build/build_docker/../src/Storages/MergeTree/MergedColumnOnlyOutputStream.cpp:42:9: error: passing result of std::move() as a const reference argument; no move will actually happen [performance-move-const-arg,-warnings-as-errors]
Jul 16 12:21:10         std::move(writer_settings),
Jul 16 12:21:10         ^~~~~~~~~~               ~
Jul 16 12:21:10 204305 warnings generated.
```

--- a/src/Databases/PostgreSQL/fetchPostgreSQLTableStructure.cpp
+++ b/src/Databases/PostgreSQL/fetchPostgreSQLTableStructure.cpp
@@ -40,7 +40,7 @@ std::set<String> fetchPostgreSQLTablesList(T & tx, const String & postgres_schem
     if (schemas.size() <= 1)
     {
         std::string query = fmt::format(
-            "SELECT tablename FROM pg_catalog.pg_tables WHERE schemaname != 'pg_catalog' AND schemaname = {}",
+            "SELECT tablename FROM pg_catalog.pg_tables WHERE schemaname = {}",
             postgres_schema.empty() ? quoteString("public") : quoteString(postgres_schema));
 
         for (auto table_name : tx.template stream<std::string>(query))
@@ -55,7 +55,7 @@ std::set<String> fetchPostgreSQLTablesList(T & tx, const String & postgres_schem
     for (const auto & schema : schemas)
     {
         std::string query = fmt::format(
-            "SELECT tablename FROM pg_catalog.pg_tables WHERE schemaname != 'pg_catalog' AND schemaname = {}",
+            "SELECT tablename FROM pg_catalog.pg_tables WHERE schemaname = {}",
             quoteString(schema));
 
         for (auto table_name : tx.template stream<std::string>(query))

